core/dbt/exceptions.py

```diff
@@ -207,22 +207,44 @@ def _fix_dupe_msg(self, path_1: str, path_2: str, name: str, type_name: str) ->
             )
 
 
-class ModelContractError(DbtRuntimeError):
+class ContractBreakingChangeError(DbtRuntimeError):
     CODE = 10016
-    MESSAGE = "Contract Error"
+    MESSAGE = "Breaking Change to Contract"
```

Comment:
I changed this, but I don't actually know where this `MESSAGE` appears. Should, this and the `type` below, be `Contract Breaking Change Error` for consistency with the exception class name?

Reply:
A few exceptions use it to construct the actual "message", but mostly it's not. Another thing we might want to clean up at some point. So I don't think it matters here.